native/sapp-windows/src/sokol_app_msvc.rs

```diff
@@ -42386,6 +42395,19 @@ extern "C" {
 extern "C" {
     pub fn glBindBuffer(target: GLenum, buffer: GLuint);
 }
+
+// TODO: make this work by linking to library that supports it
+// extern "C" {
+//     pub fn glBindBufferRange(
```

Comment:
from the docs - glBindBufferRange should be available on OpenGl 3.3
but it is not loaded in the C code - because it was not used in miniquad

if it is the only issue - you can try this fix(did not really checked, so its a blind fix): 

```
diff --git a/native/sapp-windows/external/sokol/sokol_app.h b/native/sapp-windows/external/sokol/sokol_app.h
index 4389a6e..a1ca5ff 100644
--- a/native/sapp-windows/external/sokol/sokol_app.h
+++ b/native/sapp-windows/external/sokol/sokol_app.h
@@ -3375,6 +3375,12 @@ void glBindBuffer(GLenum target, GLuint buffer) {
     _sapp_glBindBuffer(target, buffer);
 }
 
+typedef void  (GL_APIENTRY *PFN_glBindBufferRange)(GLenum target, GLuint index, GLuint buffer, GLintptr offset, GLsizeiptr size);
+static PFN_glBindBuffer _sapp_glBindBufferRange;
+void glBindBufferRange(GLenum target, GLuint index, GLuint buffer, GLintptr offset, GLsizeiptr size) {
+    _sapp_glBindBufferRange(target, index, buffer, offset, size);
+}
+
 typedef void  (GL_APIENTRY *PFN_glDeleteVertexArrays)(GLsizei n, const GLuint * arrays);
 static PFN_glDeleteVertexArrays _sapp_glDeleteVertexArrays;
 typedef void  (GL_APIENTRY *PFN_glDepthMask)(GLboolean flag);
@@ -3579,6 +3585,7 @@ _SOKOL_PRIVATE  void _sapp_win32_gl_loadfuncs(void) {
     _SAPP_GLPROC(glDisable);
     _SAPP_GLPROC(glColorMask);
     _SAPP_GLPROC(glBindBuffer);
+    _SAPP_GLPROC(glBindBufferRange);
     _SAPP_GLPROC(glDeleteVertexArrays);
     _SAPP_GLPROC(glDepthMask);
     _SAPP_GLPROC(glDrawArraysInstanced);
```

Reply:
Updated, thanks for the pointer and the code! I fixed both bindbufferrange and stencilopseparate - no more TODOs  :)